t04_refactored.py

```diff
@@ -212,7 +212,34 @@ def team_8_adv():
 
 def team_9_adv():
     pass
-    # TODO Add your code here
+   # TEAM 9
+frog = False
+question = ""
+direct = input('Enter direction: North/South/East/West')
+if direct == "North":
+    print("It's a wizard!")
+    question = input("""What do you ask the wizard(1,2,3)?
+    1)What are you doing here? 
+    2)Can you help me?
+    3)Turn me into a frog\n: """)
+    sleep(DELAY)
+if question == "1":
```

Comment:
`question` does not exist outside of the `if direct == "north":` condition. Indent this block into it.